README.md

```diff
@@ -65,45 +91,47 @@ vars:
 By default this package will build the Marketo staging models within a schema titled (<target_schema> + `_stg_marketo`) and Marketo final models within a schema titled (<target_schema> + `marketo`) in your target database. If this is not where you would like your modeled Marketo data to be written to, add the following configuration to your `dbt_project.yml` file:
```

Comment:
Reminder to update this section with the `marketo_source` default custom schema change in the source package.

Reply:
Done.